removeProperty.js

```diff
@@ -0,0 +1,21 @@
+// 4. Write the removeProperty function.
+// It should return true  if there is a property with a given key to remove.
+// Otherwise, it should return false.
+
+const user = {
+    name: 'John'
+}
+
+function removeProperty(object, key) {
+    if (object[key]) {
+        delete object[key];
+        return true;
+    }
+    return false;
+}
```

Comment:
This would not work properly in some cases.

```
const user = {
    name: 'John',
    isTall: false
}

removeProperty(user, 'isTall'); // false
```